src/Nancy/Responses/EmbeddedFileResponse.cs

```diff
@@ -47,19 +48,17 @@ private Stream GetResourceContent(Assembly assembly, string resourcePath, string
         {
             var resourceName = assembly
                 .GetManifestResourceNames()
-                .Where(x => GetFileNameFromResourceName(resourcePath, x).Equals(name, StringComparison.OrdinalIgnoreCase))
-                .Select(x => GetFileNameFromResourceName(resourcePath, x))
-                .FirstOrDefault();
+                .FirstOrDefault(x => GetFileNameFromResourceName(resourcePath, x).Equals(name, StringComparison.OrdinalIgnoreCase));
 
-            resourceName =
-                string.Concat(resourcePath, ".", resourceName);

```

Comment:
@jchannon Nope.

`Assembly.GetManifestResourceStream()` requires [case-sensitive name](https://msdn.microsoft.com/en-us/library/xc4235zt(v=vs.110).aspx) of the manifest resource, and the `FirstOrDefault()` call on line 51 returns just that: full verbatim name of the matching embedded resource.